.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: TUF-on-CI publish to GitHub Pages
+
+permissions: {}
+
+on:
+  workflow_dispatch:
```

Comment:
Should publishing also be automatic, for when staging/timestamp metadata is automatically signed?

Reply:
This is a little unintuitive but it does happen automatically: online-sign action dispatches the publish workflow if needed.

This is is built in this way in order to make "publish" the one workflow that tuf-on-ci users can customize as they want (unlike the other workflows which I'd like to consider implementation details of tuf-on-ci)

Reply:
Sounds good! Can you add a comment about that?

Reply:
I'll file an issue on tuf-on-ci to improve the comments/docs about this -- I would rather keep these workflows as direct copies of tuf-on-ci-template as much as possible to make future updates easier